packages/wonder-blocks-data/components/data.md

```diff
@@ -1,10 +1,31 @@
-The `Data` component is the client-side piece of our data architecture that
-most folks will use. It describes a data requirement in terms of a handler and
-some options. Handlers must implement the `IRequestHandler` interface.
+The `Data` component is the frontend piece of our data architecture that
+most folks will use. It describes a data requirement in terms of a handler, and
+some options. Handlers must implement the
+`IRequestHandler` interface.
 
 Among other things, the handler is responsible for fulfilling the request when
 asked to do so.
 
+#### Caching
+
+The Wonder Blocks Data framework utilizes a core internal in-memory cache for
+supporting server-side rendering (SSR) and client-side provision of data
+obtained during SSR to support hydration of the SSR result.
+
+In addition to this internal cache, a custom cache can be provided by a request
+handler. Custom caches allow for different client-side caching
+strategies (such as using local storage instead of memory). Any custom cache
+provided is ignored during SSR.
+
+When retrieving data from cache, the framework will ask the custom cache for its
+entry. If the custom cache returns an entry, that is used. If the custom cache
+returns `null`, the framework will look for a corresponding in-memory entry with
+which the framework has been initialzied, and if there, store the entry in the
+custom cache and then return it.
+
+`remove` and `removeAll` methods are also provided for removing values from the
+in-memory cache.
```

Comment:
These also exist on the custom caches as well.

Reply:
The docs are updated as the code updates across the various PRs. Perhaps I should've collated all the updates back to the first PR to make that easier to follow.